src/database/models.py

```diff
@@ -1971,168 +2041,69 @@ class Meta:
     verbose_name_plural = "HomePageSettings"
 
 
-class ActionsPageSettings(models.Model):
```

Comment:
i would not delete any of these per se.  I made all these pages different because they are mostly all not the same.  This will take off some pages and force us into doing backfills which are unnecessary in my honest opinion.

Reply:
I'm not sure whatyou meant, it doesn't take off any pages or require backfills.  In fact all the page settings models except HomePageSettings were identical, except for DonatePageSettings which had one additional field; I added that. I wanted to take advantage of the commonality of page settings to add the optional social media links and images